content/docs/3.0/concepts/glossary.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Glossary
+summary: Glossaries about TiKV.
+menu:
+    docs:
+        parent: Concepts
```

Comment:
Seems reference is more appropriate

Reply:
Actually I think these are some good concept material (reference, too) which users may want to notice easily when they are learning about TiKV.